install.sh

```diff
@@ -1,21 +1,52 @@
 #!/bin/sh -e
 
-### Stage 1 - Prerequisites
+### SETUP
 
-# constants for helm_operator
+# constants for helm-operator
 helm_operator_version=1.0.0
 helm_operator_namespace=md-helm-operator
 helm_operator_replica_count=1
 helm_versions=v3
 
-# CERT-MANAGER CRDs
+# build the --set-files flag for helm
+
+values_from_files=""
+function setfile() {
+  if [ -z "$values_from_files" ]
+  then
+    values_from_files=$1
+  else
+    values_from_files="$values_from_files,$1"
+  fi
+}
+
+setfile ca.crt=config/pki/ca.crt
+setfile etcd.ca.crt=config/pki/etcd/ca.crt
+setfile etcd.healthcheckClient.crt=config/pki/etcd/healthcheck-client.crt
+setfile etcd.healthcheckClient.key=config/pki/etcd/healthcheck-client.key
+
+### GENERATE THE TEMPLATE OUTPUTS
+
+# create tmp directory if it does not exist
+mkdir -p tmp 
+
+# template out the chart 
+helm template . \
+  --set-file $values_from_files \
```

Comment:
This is the new flag. The other parts of this section are just moved from near the end of the script to near the top so that if templating fails, the script will halt before installing prerequisites.

Reply:
seems to work. You do have to have the keys in the values file, even if they are just blank.

Reply:
I could swear I deleted mine entirely?

Reply:
If this is true I'ma fix it before merging.